tests/test_renderer_style.py

```diff
@@ -1,45 +1,18 @@
-from markdown_it import MarkdownIt
+from pathlib import Path
+
+from markdown_it.utils import read_fixture_file
 import pytest
 
-from mdformat.renderer import MDRenderer
+from mdformat import text as render
 
-STYLE_CASES = (
-    {
-        "name": "strip paragraph lines",
-        "input_md": "trailing whitespace \n"
-        "at the end of paragraph lines \n"
-        "should be stripped                   \n",
-        "output_md": "trailing whitespace\n"
-        "at the end of paragraph lines\n"
-        "should be stripped\n",
-    },
-    {
-        "name": "strip quotes",
-        "input_md": "> Paragraph 1\n" "> \n" "> Paragraph 2\n",
-        "output_md": "> Paragraph 1\n" ">\n" "> Paragraph 2\n",
-    },
-    {
-        "name": "no escape ampersand",
-        "input_md": "R&B, rock & roll\n",
-        "output_md": "R&B, rock & roll\n",
-    },
-    {
-        "name": "list whitespaces",
-        "input_md": "- item one\n  \n- item two\n  - sublist\n    \n  - sublist\n",
-        "output_md": "- item one\n\n- item two\n\n  - sublist\n\n  - sublist\n",
-    },
-    {
-        "name": "convert setext to ATX heading",
-        "input_md": "Top level heading\n=========\n\n2nd level heading\n---------",
-        "output_md": "# Top level heading\n\n## 2nd level heading\n",
-    },
-)
+STYLE_CASES = read_fixture_file(Path(__file__).parent / "data" / "fixtures.md")
 
 
-@pytest.mark.parametrize("entry", STYLE_CASES, ids=[c["name"] for c in STYLE_CASES])
-def test_renderer_style(entry):
+@pytest.mark.parametrize(
+    "line,title,text,expected", STYLE_CASES, ids=[f[1] for f in STYLE_CASES]
+)
+def test_renderer_style(line, title, text, expected):
     """Test Markdown renderer renders expected style."""
-    md_original = entry["input_md"]
-    md_new = MarkdownIt(renderer_cls=MDRenderer).render(md_original)
-    expected_md = entry["output_md"]
-    assert md_new == expected_md
+    md_new = render(text)
+    print(md_new)
```

Comment:
Should we remove the print?

Reply:
I'm going to leave it in, but only when the assertion fails, since it's a very handy practice for adding/modifying tests, to add a new test input, run pytest, then copy/paste the printed output to the expected section.